plugin/context.go

```diff
@@ -127,6 +127,19 @@ func (ps *Set) Get(t Type) (interface{}, error) {
 	return nil, fmt.Errorf("no plugins registered for %s: %w", t, ErrPluginNotFound)
 }
 
+// GetByID returns the plugin of the given type and ID
```

Comment:
Maybe we have a chance to clean up this interface with the package move. We should really have `Get(t Type, id string)` and existing Get would be `GetFirst` or just move away from that approach.

Reply:
What do you think about delaying the change until the package move?  Otherwise we'll have other, unrelated changes in this PR to deal with the refactoring.

Reply:
Yes, we can do that in the other repo